pyproject.toml

```diff
@@ -32,13 +32,12 @@ mypy = "^0.910"
 isort = "^5.9.3"
 pytest-sugar = "^0.9.4"
 black = "^21.7b0"
-flake8 = "^3.9.2"
 python-dotenv = "^0.19.0"
 pytest-cov = "^2.12.1"
 uvicorn = "^0.15.0"
 loguru = "^0.5.3"
 Sphinx = "^4.2.0"
-sphinx-view = "^0.3.0"
+pyproject-flake8 = "^0.0.1-alpha.2"
```

Comment:
This version bothers me... I don't think we should rely on alpha quality tools. Lets wait and see if this projects gets a stable version.

Reply:
This version of the tool functions in its current version. It's also only supposed to be a dev dependency (I added it in the wrong place), and has no effect on the operation of the library. If something goes wrong or the tool falls out of maintenance, we can revert to the prior way of using flake8.

Reply:
for me it's ok